libvalkeycluster/tests/ct_commands.c

```diff
@@ -202,23 +203,23 @@ void test_eval(redisClusterContext *cc) {
     // Error handling in EVAL
 
     // Request without key, fails due to no key given.
-    reply = (redisReply *)redisClusterCommand(
-        cc, "eval %s 0", "return redis.call('set','foo','bar')");
+    reply = (valkeyReply *)valkeyClusterCommand(
+        cc, "eval %s 0", "return valkey.call('set','foo','bar')");
```

Comment:
The test is passing when running when using Valkey 7.2.5
Maybe its changed on unstable?

Reply:
I might also be wrong about it :)

Reply:
We don't have `valkey.call`. Use `server.call` (or `redis.call` which still works for older versions if we care about those).

Reply:
Lets keep redis.call for now since then we can run tests towards redis in CI.
We will have to add some logic here to determine which server the tests run towards.
There is already a function for getting the server version.